Karma/Models/Post.cs

```diff
@@ -45,15 +45,15 @@ public static string GetPostTypeName(bool isDonation)
             return isDonation ? "Offer" : "Request";
         }
 
-        // By visibility and by date (decreasing order).
+        // By visibility and by date (descending order).
         public int CompareTo(Post other)
         {
             if (other.IsVisible == false && this.IsVisible == true)
-                return 1;
-            else if (other.IsVisible == true && this.IsVisible == false)
                 return -1;
+            else if (other.IsVisible == true && this.IsVisible == false)
```

Comment:
Sorry that I didn't ask this in the last PR but I don't really understand why would we need to compare something by visibility? I guess posts that are not visible should not be shown at all, so there is no reason to implement sorting logic according to the visibility attribute?

Reply:
Okay, sounds good.